src/resources/fine-tuning/jobs.ts

```diff
@@ -192,6 +192,8 @@ export interface FineTuningJobEvent {
   message: string;
 
   object: string;
+
+  id: string;
```

Comment:
This should be autogenerated by the SDK, updating in our spec and regenerating separately. cc @rattrayalex

Reply:
➕ 

If the types here are incorrect, can you update here? https://github.com/openai/openai-openapi/blob/master/openapi.yaml

Reply:
Should I wait until this is updated in master and then pull to merge?